src/main/java/Duke.java

```diff
@@ -1,10 +1,312 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
+    private final String VERSION_NUMBER = "1.0.0";
+    private final Scanner SC = new Scanner(System.in);
+    private final Path DUKE_DATA_FILE_PATH = Paths.get("data", "duke.txt");
+    private final Path DUKE_DATA_DIR_PATH = Paths.get("data");
+    private final String NEW_LINE = "\n";
+    private final String HORIZONTAL_LINE =
+            "    ____________________________________________________________";
+    private final String PADDING = "      ";
+
+    private enum Commands {
+        EXIT("BYE"), DELETE("DELETE "), LIST("LIST"),
+        DONE("DONE "), TODO("TODO "), EVENT("EVENT "),
+        DEADLINE("DEADLINE "), DELETEALL("DELETE ALL");
```

Comment:
Should enum names be written in PascalCase instead?

Reply:
The enum class Commands is in PascalCase, the values in the enum class are constants and should be in UPPER_CASE

https://nus-cs2103-ay2021s1.github.io/website/se-book-adapted/chapters/oop.html#enumerations